README.md

```diff
@@ -22,13 +22,16 @@
 **Kakarot** is an Ethereum Virtual Machine written in Cairo. It means it can be
 deployed on StarkNet, a layer 2 scaling solution for Ethereum, and run an EVM
 bytecode program. Hence, Kakarot can be used to run Ethereum smart contracts on
-StarkNet. Kakarot is the super sayajin ZK-EVM ! Why ? Because:
-`It's over 9000!!!!!`. It is a work in progress, and it is not ready for
-production.
+StarkNet. Kakarot is the super sayajin zkEVM ! Why ? Because:
```

Comment:
I think english doesn't have spaces between ponctuation like exclamation and interrogation marks